docs/reach/rs2/reachview/mobile-data.md

```diff
@@ -10,6 +10,9 @@ Enable Mobile data on Reach to [get corrections from NTRIP over the Internet](ht
 
 	</center>
 
+!!! note ""
```

Comment:
Sorry if I wasn't clear enough. We don't need to turn it into a note, we need to move this link below the note about APN details. Also, please make sure you fixed the link as I asked.